src/components/SearchModal/index.tsx

```diff
@@ -184,6 +184,19 @@ function SearchModal({
   // if the current input is an address, and we don't have the token in context, try to fetch it
   const token = useToken(searchQuery)
   const [temporaryToken, setTemporaryToken] = useState<Token | null>()
+
+  // filters for ordering
+  const [activeFilter, setActiveFilter] = useState(FILTERS.BALANCES)
+
+  // toggle specific token import view
+  const [showTokenImport, setShowTokenImport] = useState(false)
+
+  // for sorting
+  const escapeStringRegexp = string => string
```

Comment:
this should be: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Escaping

Reply:
let's just make that a util function, make sure we're not doing this anywhere else, and remove the `escape-string-regexp` dependency

Reply:
Yep adding this to utils